scripts/workspace-status.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+echo "STABLE_GIT_COMMIT $(git rev-parse HEAD)"
```

Comment:
This will not detect dirty git repo. Perhaps we need to combine with `git status --porcelain` to detect if the repo is clean and the HEAD commit is printed, otherwise an empty string is printed, which should be handled similar. to an empty version

Reply:
Done